app/views/stages/_fields.html.erb

```diff
@@ -61,6 +61,14 @@
     </div>
   </div>
 
+  <div class="form-group">
+    <%= form.label :name, "Template Environment", class: "col-lg-2 control-label" %>
+    <div class="col-lg-4">
+      <%= form.select :template_environment_id, Environment.all.collect {|e| [e.name, e.id] }, { include_blank: true }, {class: 'form-control' } %>
```

Comment:
I don't think we have any stages that belong to multiple environments ... so not really useful ?